config/routes.rb

```diff
@@ -8,5 +8,7 @@
 
   get '/signup', to: 'users#new', as: :new_user_path
 
+  get  '/create_bet', to: 'bets#new', as: :new_bet
```

Comment:
You should be using the `resources` syntax in order to create RESTful routes. 

http://guides.rubyonrails.org/routing.html